ci/docker/connector_test/Dockerfile

```
@@ -3,13 +3,14 @@ ARG BASE_IMAGE
 FROM $BASE_IMAGE
 
 # This is to solve permission issue, read https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
-RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.11/gosu-amd64"
+ARG GOSU_URL
```

Keeping default values can make sure that we don't break downsstream project that might be using this Dockerfile
```suggestion
ARG GOSU_URL=https://github.com/tianon/gosu/releases/download/1.11/gosu-amd64
```